src/app/components/drawer/Projects/NewArticleButton.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import ButtonMain from '../../cds/buttons-checkboxes-chips/ButtonMain';
+import AddIcon from '../../../icons/add.svg';
+import styles from './NewArticleButton.module.css';
+
+const NewArticleButton = ({ options, onMenuItemClick }) => {
+  const [anchorEl, setAnchorEl] = React.useState(null);
+
+  return (
+    <React.Fragment>
+      <ButtonMain
+        variant="contained"
+        size="default"
+        theme="text"
+        iconLeft={<AddIcon />}
+        onClick={e => setAnchorEl(e.currentTarget)}
+        label="New Article"
+        buttonProps={{
+          id: 'new-article-menu__open-button',
+        }}
+        customStyle={{
```

Comment:
@danielevalverde because of the design system color changes I wouldn't worry too much about matching the designs 100% right now.
So I would remove these custom styles. We can focus on the functionality right now, and I can help clean up colors as I clean up figma